### ISSUE TITLE
Random Order Generated.

### ISSUE THREAD

**Issue by adityarudrawar** [krishnavelu/alice_blue#261]
A random order of Reliance was generated in my account which is not visible in my ORDERBOOK but its showing me in the API: the oms_order_id = '210225000118903'. This gives me the order history when I call alice.get_order_history('210225000118903').

but when I get the orders by alice.get_order_history() that oms_order_id is not visible.

**Comment by krishnavelu** [comment]
where did you got the order_id '210225000118903' from?

**Comment by adityarudrawar** [comment]
I am storing logs of each order using the api function get_order_history() it showed me this id generated at time 13:09:28 but that order is not showing in my web portal order book. When I checked the that an extra order is placed I looked at the logs and called get_order_history() again to my surprise it didn't show that order id again, but when I called history of that specific order by passing the oms_order_id. It showed the full logs

**Comment by krishnavelu** [comment]
I don’t know where from it came. What’s your problem exactly?

**Comment by adityarudrawar** [comment]
The problem: my logged files showed an order was generated with the mentioned oms_order_id which I didn't placed. I was able to call the history by passing that if onto the get_order_history() but When i called the whole order book its NOT showing that order information

**Comment by krishnavelu** [comment]
What’s the problem in that?

**Comment by adityarudrawar** [comment]
I DID NOT place that order.

**Comment by krishnavelu** [comment]
So?

**Comment by adityarudrawar** [comment]
When I downloaded the whole order book it didn't show any information about that order, NOTHING. but I was able to get the info by passing id to get_order_history(order id). How does that happe
ened?

**Comment by krishnavelu** [comment]
I don’t know how did you store or when did you store. It might be from previous day’s execution. Create issue only if you found anything wrong with this library. I cannot help anything about your application.

**Comment by adityarudrawar** [comment]
I am NOT talking about my application. I am storing logs of every order generated in my account by repetitive calling of get_order_history(). On 25Feb at 13:09 that order was seen in the logs I am ONLY placing orders through alice blue web portal and not through API. It showed an order of Reliance was generated with that id. I didn't place that ORDER. so I downloaded the whole order book and it didn't show any info about that order. So how was that order id was not their in my order book but I was able to get info of that order
